docs/Performance/promise.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: default
+title: Promise
+name: Overview
+parent: Performance
+nav_order: 1
+---
+
+## Promise
+
+If you are new to Promises, I would suggest to make yourself familiar before continuing:
```

Comment:
please use 3rd person language à la "If you are new to Promises, please make yourself familiar ..."

Reply:
added in one of the latest commits